clang/lib/Format/MatchFilePath.cpp

```diff
@@ -49,11 +49,29 @@ bool matchFilePath(StringRef Pattern, StringRef FilePath) {
         return false;
       break;
     case '*': {
-      while (++I < EOP && Pattern[I] == '*') { // Skip consecutive stars.
+      if (I + 1 < EOP && Pattern[I + 1] == '*') {
+        // Handle '**' pattern
```

Comment:
```suggestion
        // Handle '**' pattern.
```
Always end in full stop.